content/en/serverless/configuration/_index.md

```diff
@@ -143,7 +143,7 @@ Datadog can also enrich the collected telemetry with existing AWS resource tags
 
 ## Collect the request and response payloads
 
-<div class="alert alert-info">This feature is currently supported for Python, Node.js, Java, and .NET.</div>
+<div class="alert alert-info">This feature is currently supported for Python, Node.js, and .NET.</div>
```

Comment:
any doc other than `en`?

Reply:
I don't speak Japanese, but I think here as well? https://github.com/DataDog/documentation/blob/master/content/ja/serverless/configuration/_index.md#%E3%83%AA%E3%82%AF%E3%82%A8%E3%82%B9%E3%83%88%E3%81%A8%E3%83%AC%E3%82%B9%E3%83%9D%E3%83%B3%E3%82%B9%E3%81%AE%E3%83%9A%E3%82%A4%E3%83%AD%E3%83%BC%E3%83%89%E3%82%92%E5%8F%8E%E9%9B%86%E3%81%99%E3%82%8B

Reply:
I believe we have a bot that automatically translates this though. Maybe someone from the documentation team can help clarify that

Reply:
<img width="1165" alt="image" src="https://user-images.githubusercontent.com/47579703/176202587-03bef2f8-1b45-4863-b31a-faa8541f97ad.png">

The translation bot is `guacbot`. So all good!
https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958063/Pull+Requests#Translations-(guacbot)%3A

Reply:
It delights me you used my Udemy talk to learn this! Correct, the updates to JA and FR will be handled by the translation tools.